api/src/services/purged-docs.js

```diff
@@ -1,13 +1,14 @@
 const db = require('../db');
+const dbWatcher = require('./db-watcher');
 const environment = require('../environment');
 const purgingUtils = require('@medic/purging-utils');
-const cacheService = require('./cache');
```

Comment:
I think this is the only use of the cacheService so we can drop it and the node-cache package.

Reply:
Cool, I considered dropping the package too.